runtime/stdlib/test.go

```diff
@@ -549,6 +562,76 @@ var testExpectFunction = interpreter.NewUnmeteredHostFunctionValue(
 	},
 )
 
+// Test.expectFailure function
+const expectFailureFunctionDocString = `
+Wraps a function call in a closure, and expects it to fail with a certain error message.
+`
+
+const expectFailureFunctionName = "expectFailure"
+
+var expectFailureFunctionType = &sema.FunctionType{
+	Parameters: []sema.Parameter{
+		{
+			Label:      sema.ArgumentLabelNotRequired,
+			Identifier: "functionWrapper",
+			TypeAnnotation: sema.NewTypeAnnotation(
+				&sema.FunctionType{
+					Parameters: []sema.Parameter{},
+					ReturnTypeAnnotation: sema.NewTypeAnnotation(
+						sema.VoidType,
+					),
+				},
+			),
+		},
+		{
+			Label:      sema.ArgumentLabelNotRequired,
+			Identifier: "errorMessage",
+			TypeAnnotation: sema.NewTypeAnnotation(
+				sema.StringType,
+			),
+		},
+	},
+	ReturnTypeAnnotation: sema.NewTypeAnnotation(
+		sema.VoidType,
+	),
+	RequiredArgumentCount: sema.RequiredArgumentCount(2),
+}
+
+var expectFailureFunction = interpreter.NewUnmeteredHostFunctionValue(
+	expectFailureFunctionType,
+	func(invocation interpreter.Invocation) interpreter.Value {
+		inter := invocation.Interpreter
+		functionValue := invocation.Arguments[0].(interpreter.FunctionValue)
```

Comment:
```suggestion
		functionValue, ok := invocation.Arguments[0].(interpreter.FunctionValue)
		if !ok {
			panic(errors.NewUnreachableError())
		}
```

Reply:
Fixed in https://github.com/onflow/cadence/pull/2468/commits/136bc2a56bfa322ae2086f7bc6fa1027090276c2